aws/iam/policy/NextstrainDotOrgServerInstanceDev.json

```diff
@@ -43,6 +43,18 @@
         "arn:aws:s3:::nextstrain-groups/test/*",
         "arn:aws:s3:::nextstrain-groups/test-private/*"
       ]
+    },
```

Comment:
Comparing this JSON with the current IAM Policy (what the IAM console calls "version 7"), blocks are missing for `"Sid": "ListInventories"` and `"Sid": "CognitoUserPoolActions"`. These are flagged up as blocks to remove when I run `terraform -chdir=env/testing/ plan` so I'm curious (as a terraform beginner) to know why it wasn't flagged up for you? 

(P.S. I added `"Sid": "ListInventories"` at the same time as `"Sid": "GetInventories"`, but I didn't add `"Sid": "CognitoUserPoolActions"`.)

Reply:
Oh - I just noticed they are part of your (orphaned) commit ff5599774ae22ad38338b73ec7d2890c23fe1f94!

Reply:
Also worth noting that I'll probably remove most of these permissions from the dev server policy as we're now no longer listing / getting the actual S3 inventories from the server process. Instead I'll create a new IAM user to be used by whichever service we use to run `./manifest/main.js` (which produces the `resources.json.gz` file)

Reply:
The current active policy that you see on AWS console is a combination of:

1. a00b8e11941e115ad14d314d7a4c8c50388729b9 from #700
2. 0aaf7fa94bf4f81748e95393e7102945f0739b04 from this PR
3. ad853a2445d6fd726ada036b73cc05608cf00011 from #581

I broke it up so it would be obvious which policy changes are the minimum necessary for each PR, as it's not clear what order the PRs will get merged. However, a combination of all the changes must be applied for all review apps to work properly.

This comes down to a limitation with the way we're sharing AWS resources across Heroku review apps. It's not a problem caused by Terraform adoption since, prior to that, these policy JSONs were already tracked manually, but it makes Terraform adoption harder in that, to `terraform apply` without removing changes that are made in other unmerged PRs, we have to [cherry-pick those changes](https://github.com/nextstrain/nextstrain.org/pull/581#discussion_r1344912988) before applying.

I think to do it right, we'd have a separate IAM user/policy auto-generated for each review app

Reply:
> Also worth noting that I'll probably remove most of these permissions from the dev server policy as we're now no longer listing / getting the actual S3 inventories from the server process.

@jameshadfield Ok for me just to revert these out-of-band additions as part of deploying changes for #581?

> Instead I'll create a new IAM user to be used by whichever service we use to run `./manifest/main.js` (which produces the `resources.json.gz` file)

I'd hold off on the IAM user until we decide where to place that process. If it's on GitHub Actions, for instance, then we'd prefer to use assumed roles (per https://github.com/nextstrain/private/issues/22).

The role and associated policy itself should be part of this repo's Terraform configuration, though, so it's managed alongside the code requiring it.

Reply:
> Ok for me just to revert these out-of-band additions

The only permission I'd like to keep (for the dev policy) is GET access to `resources.json.gz`

Reply:
https://github.com/nextstrain/nextstrain.org/pull/719#issuecomment-1765165740